spiffe-helper.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: spiffe-helper
+  version: 0.9.0
+  epoch: 0
+  description: A helper utility for SPIFFE (Secure Production Identity Framework For Everyone) operations.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0" # Required to build a fully static binary
```

Comment:
we don't want to build a fully static binary unless it's really required. For a user perspective, it's convenient but form a distribution perspective what this means is that we are vendoring dependencies which might have CVEs down the line so I'll consider removing it if it's building without it.